tools/cvm-image-rewriter/README.md

```diff
@@ -76,7 +77,7 @@ Please refer [the sample plugin](/tools/cvm-image-rewriter/pre-stage/99-test/).
 1. This tool has been tested on `Ubuntu 22.04` and `Debian 10`. It is recommend to use
 `Ubuntu 22.04`.
 
-2. This tool can run on bare metal or virtual machine (with nest VM like `Intel VT-x`, detailed in [Section 3.4](#3.4-Run-in-Nested-VM-(Optional)))
+2. This tool can run on bare metal or within a virtual machine using nesting as detailed in [Section 3.4](#3.4-Run-in-Nested-VM-(Optional))
 
 3. Please install following packages on Ubuntu/Debian:
```

Comment:
I would reword this as  "Please install the following"

Reply:
For 1 through 7 some of these end with a period and some do not. I would make it consistent.

Reply:
Further below in number 7 I would change it to   "If it is needed to customize the connection URL, you can"

Reply:
Also, I would change this and make it its own sentence "Please make sure the current user belongs to the libvirt group via the following commands"